### DISCUSSION TITLE
Is there any way to test Deep Selectors (`:deep()`) ?

### DISCUSSION THREAD

**Discussion by webfansplz** [vuejs/test-utils#2164] [Q&A]
```vue
<style scoped>
.a :deep(.b) {
  /* ... */
}
</style>
```

Is there any way or workaround to test the `:deep()` is be applied correctly to the element ?  🙏

**Comment by freakzlike** [comment]
I would use some E2E real browser testing (like Cypress) and check if the style has been applied to the element

**Reply by webfansplz** [reply]
Yep, seems e2e is the only way to cover it.
